reconcile/jenkins_job_builds_cleaner.py

```diff
@@ -83,11 +83,12 @@ def run(dry_run):
             )
 
         token = instance["token"]
+        instance_name = instance.get("name")
```

Comment:
the instance is supposed to always have a name, so you could use `instance["name"]` and let the code fail if that would be not the case (for whatever reason)

Reply:
just a nit - feel free to ignore